src/Toml/Bi/Combinators.hs

```diff
@@ -513,3 +513,33 @@ map keyCodec valCodec key = Codec input output
                     insertTableArrays key $ t :| (ts ++ tomls)
 
         dict <$ modify updateAction
+
+tableMapCodec
+    :: Ord key
+    => TomlBiMap Key key  -- ^ Bidirectional converter between TOML and Map keys
+    -> TomlBiMap val AnyValue  -- ^ Bidirectional converter between TOML and Map values
+    -> Key  -- ^ Table name key
+    -> TomlCodec (Map key val)
+tableMapCodec keyBiMap valBiMap key = Codec input output
+  where
+    -- input :: Env (Map key val)
```

Comment:
According to our style guide, it's recommended to write explicit type signatures for functions inside `where`. I believe, you tried doing this but got a compiler error. To fix this error, you need to write type signatlure with explicit `forall`, like this:

```haskell
:: forall key val . Ord key => ...
```

```suggestion
    input :: Env (Map key val)
```